src/psyclone/domain/lfric/lfric_kern.py

```diff
@@ -435,11 +443,7 @@ def last_cell_all_colours_symbol(self):
             raise InternalError(f"Kernel '{self.name}' is not inside a "
                                 f"coloured loop.")
 
-        if self._is_intergrid:
-            if self._intergrid_ref is None:
-                raise InternalError(
-                    f"Colourmap information for kernel '{self.name}' has "
-                    f"not yet been initialised")
+        if self.is_intergrid:
```

Comment:
Please update the 'raises' bit of the docstring.

Reply:
Done